src/anemoi/models/interface/__init__.py

```diff
@@ -96,8 +97,15 @@ def _build_model(self) -> None:
             config=self.config, data_indices=self.data_indices, graph_data=self.graph_data
         )
 
-        # Use the forward method of the model directly
-        self.forward = self.model.forward
+    def forward(self, x: torch.Tensor, model_comm_group: Optional[ProcessGroup] = None) -> torch.Tensor:
```

Comment:
Wouldn't it make sense to have this better visible, e.g. in the name here? Even if we haven't the alternativ implemented at the moment (but which makes sense to have for anemoi, e.g. with obs the residual is typically not possible).

Reply:
I agree, I think instead of `self.tendency_mode` there should be a `self.prediction_mode` which can take the values 'state', 'residual' or 'tendency'. I will try to change that when we open the PR with anemoi-training.